.mint/release.yml

```diff
@@ -172,7 +176,7 @@ tasks:
   - key: publish-production-release
     use: [install-gh-cli, code]
     after: [extract-version-details, build-and-upload-binaries]
-    if: ${{ init.kind == "production" }}
+    if: ${{ init.kind == "production" && tasks.build-and-upload-binaries.tasks.build-and-upload-linux-amd64-binaries.values.verified-linux-amd64-binary == "true" }}
```

Comment:
Should we fail in this case? Right now this will appear successful

Reply:
Good call. I made it fail.

Reply:
I originally made it fail in a23ab8baa6588fcd55da70a00039ea05a6a9bb03, but then I realized that logic branch was in the wrong task since we could still proceed with updating aliases. I decided to change the logic branch entirely in 5f51e8f0cacfdb1a9a76e43ddbe3fa04da31ca74. I originally wanted to check the value to ensure the logic branch was hit and that there wasn't a typo or misconfiguration in the logic branch. Simplifying the conditional though and making it opt-out instead of opt-in feels sufficient and simpler though.